plugins/processors/awsapplicationsignals/internal/resolver/attributesresolver.go

```diff
@@ -0,0 +1,162 @@
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: MIT
+
+package resolver
+
+import (
+	"context"
+	"errors"
+	"fmt"
+	"strings"
+
+	"go.opentelemetry.io/collector/pdata/pcommon"
+	semconv1 "go.opentelemetry.io/collector/semconv/v1.17.0"
```

Comment:
Why are we using v1.17.0 here and v1.18.0 in other packages?

Reply:
Changed the import.